docs/gestor_dependencias.md

```diff
@@ -0,0 +1,32 @@
+# Gestor de dependencias
+## Opciones
+Durante el objetivo 2, el encargado de trabajar en el repositorio usó [pyproject.toml](../pyproyect.toml) como archivo de configuración para el desarrollo del proyecto, por ello se elegirá un gestor que sea compatible con el mismo. Algunas de las opciones más comunes son:
+
+### 1. pip
+* Gestor de paquetes de Python y viene preinstalado con la mayoría de las instalaciones de Python.
+* pip sólamente soporta parcialmente pyproject.toml, ya que reconoce este archivo gracias a la PEP 518 (que define pyproject.toml para definir configuraciones de construcción de paquetes). Sin embargo, pip no maneja las dependencias directamente desde pyproject.toml sin ayuda adicional
+
+### 2. Poetry
+* Poetry es un gestor de dependencias y un manejador de entornos que permite gestionar dependencias, versiones de Python y la publicación de paquetes. Es popular en proyectos modernos que necesitan un control más preciso de las versiones de los paquetes.
```

Comment:
Esto no es un PDF impreso que hay que entregar. ¿Os cuesta mucho trabajo enlazar las herramientas que ponéis?

Reply:
Modificado en [d80aac0](https://github.com/JLombar/HorariosAutomatricula/pull/15/commits/d80aac08c217368f92c699aa6f779d6000064bf2), también se han añadido sobre el gestor de tareas en [aaf042f](https://github.com/JLombar/HorariosAutomatricula/pull/15/commits/aaf042f6cc95eb0dad13b578a3189916e1f8424e)